packages/wonder-blocks-dropdown/components/single-select.js

```diff
@@ -271,24 +345,41 @@ export default class SingleSelect extends React.Component<Props, State> {
     }
 
     render() {
-        const {alignment, dropdownStyle, light, style} = this.props;
-        const items = this.getMenuItems();
-        const opener = this.renderOpener(items.length);
+        const {
+            alignment,
+            children,
+            dropdownStyle,
+            isFilterable,
+            light,
+            style,
+        } = this.props;
+        const {searchText} = this.state;
+        const allChildren = React.Children.toArray(children).filter(Boolean);
+        const filteredItems = this.getMenuItems(allChildren);
```

Comment:
style nit: I'd probably move `React.Children.toArray(children).filter(Boolean)` inside of `this.getMenuItems()`.  It groups all of the children logic together instead of spreading it across multiple methods.

Reply:
I added it here because we are using allChildren in two different methods (`getMenuItems` and `renderOpener`). What do you think if we keep it that way to prevent calling `React.Children.toArrray` in two different places?

Reply:
I didn't realize that.  Let's keep it as is.